lib/active_fedora/attributes.rb

```diff
@@ -49,21 +45,10 @@ def []=(key, value)
         # The attribute is stored in the RDF graph for this object
         send(key.to_s + "=", value)
       else
-        # The attribute is a delegate to a datastream
-        array_setter(key, value)
+        raise ArgumentError, "Unknown attribute #{key}"
       end
     end
 
-    # @return [Boolean] true if there is an reader method and it returns a
-    # value different from the new_value.
-    def value_has_changed?(field, new_value)
-      new_value != array_reader(field)
-    end
-
-    def mark_as_changed(field)
```

Comment:
@jcoyne No deprecation warnings for this method or value_has_changed.  Was this intentional?